src/Component/dummyData.tsx

```diff
@@ -0,0 +1,81 @@
+export interface actor {
```

Comment:
```suggestion
export interface Actor {
```

Reply:
Also move out the types to a separate file. `Actor` is an important type and will be used in a lot of components. Defining it in a file named `dummyData` is a bad idea. The file will not live for longer.